spec/asyncapi.md

```diff
@@ -1882,6 +1874,37 @@ components:
             maximum: 100
 ```
 
+#### <a name="multiFormatSchemaObject"></a>Multi Format Schema Object
+
+The Multi Format Schema Object allows to define non AsyncAPI schema.
+
+##### Fixed Fields
+
+Field Name | Type | Description
+---|:---:|---
+<a name="schemaObjectSchemaFormat"></a>schemaFormat | `string` | **Required**. A string containing the name of the schema format used to define the message payload. If omitted, implementations should parse the payload as a [Schema object](#schemaObject). When the payload is defined using a `$ref` to a remote file, it is RECOMMENDED the schema format includes the file encoding type to allow implementations to parse the file correctly. E.g., adding `+yaml` if content type is `application/vnd.apache.avro` results in `application/vnd.apache.avro+yaml`.<br/><br/>Check out the [supported schema formats table](#multiFormatSchemaFormatTable) for more information. Custom values are allowed but their implementation is OPTIONAL. A custom value MUST NOT refer to one of the schema formats listed in the [table](#multiFormatSchemaFormatTable).<br/><br/>When using a [Reference Object](#referenceObject) within the schema, the `schemaFormat` of the reference target SHOULD be exactly the same as of the `schema` containing the reference.
+<a name="schemaObjectSchema"></a>schema | `any` | **Required**. Definition of the message payload. It can be of any type but defaults to [Schema Object](#schemaObject). It must match the schema format, including encoding type - e.g Avro should be inlined as either a YAML or JSON object NOT a string to be parsed as YAML or JSON. - Non-JSON based schema, e.g. Protobuff or XSD should be inlined as string.
```

Comment:
```suggestion
<a name="multiFormatSchemaObjectSchema"></a>schema | `any` | **Required**. Definition of the message payload. It can be of any type but defaults to [Schema Object](#schemaObject). It MUST match the schema format defined in [`schemaFormat`](#multiFormatSchemaObjectSchemaFormat), including the encoding type. E.g., Avro should be inlined as either a YAML or JSON object instead of as a string to be parsed as YAML or JSON. Non-JSON-based schemas (e.g., Protobuf or XSD) MUST be inlined as a string.
```